tests/test_association_db_sqlite.py

```diff
@@ -29,13 +29,14 @@
     AssociationDBSqliteTask, \
     AssociationDBSqliteConfig, \
     make_minimal_dia_source_schema
+from lsst.ap.association.assoc_afw_utils import \
```

Comment:
Again, probably better to `from lsst.ap.association import afwUtils` and then you'll have the whole namespace.